src/python-common/ceph/deployment/inventory.py

```diff
@@ -60,6 +62,7 @@ def __init__(self,
         self.lvs = lvs
         self.device_id = device_id
         self.lsm_data = lsm_data if lsm_data is not None else {}  # type: Dict[str, Dict[str, str]]
+        self.created = datetime_now()
```

Comment:
note that always setting this to `datetime_now()` means this will be reset upon mgr failover even if the device has not actually been refreshed yet. If we want to actually keep the refreshed field accurate across failovers you would need something like `self.created = created if created is not None else datetime_now()` as well as fixing up to_json and from_json to include and work with it (datetime objects are not json serializable, see what we have to do to make it work elsewhere https://github.com/ceph/ceph/blob/master/src/pybind/mgr/cephadm/inventory.py#L664 and https://github.com/ceph/ceph/blob/master/src/pybind/mgr/cephadm/inventory.py#L473).